tools/rapids-rattler-channel-string

```diff
@@ -2,20 +2,27 @@
 
 RAPIDS_CHANNEL="rapidsai-nightly"
 DASK_CHANNEL="dask/label/dev"
+NVIDIA_CHANNEL="nvidia"
 
 # Replace dev/nightly channels if build is a release build
 if rapids-is-release-build; then
   RAPIDS_CHANNEL="rapidsai"
   DASK_CHANNEL=""
 fi
 
-channels=("$RAPIDS_CHANNEL" "$DASK_CHANNEL" "conda-forge" "nvidia")
+# Only include nvidia channel if we're on cuda11
+IFS='.' read -ra CUDA_VERSION_ARRAY <<< "$RAPIDS_CUDA_VERSION"
+if [ "${CUDA_VERSION_ARRAY[0]}" -eq "12" ]; then
+  NVIDIA_CHANNEL=""
+fi
```

Comment:
Can you flip this, like this?

```shell
NVIDIA_CHANNEL=""
if [ "${CUDA_VERSION_ARRAY[0]}" -eq "11" ]; then
  NVIDIA_CHANNEL="nvidia"
fi
```

That way, it'd work in a future with newer major CUDA versions than 12.

Reply:
yep, that's a good shout -- flipped